src/main/java/be/syntra/devshop/DevshopBack/repositories/CartRepository.java

```diff
@@ -1,10 +1,10 @@
 package be.syntra.devshop.DevshopBack.repositories;
 
-import be.syntra.devshop.DevshopBack.entities.Cart;
+import be.syntra.devshop.DevshopBack.entities.ShopOrder;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
 @Repository
-public interface CartRepository extends JpaRepository<Cart, Long> {
+public interface CartRepository extends JpaRepository<ShopOrder, Long> {
```

Comment:
yeah, here I draw the line though :D better rename this to ShopOrderRepository

Reply:
hehehe, you're right, fixed